libavcodec/vvc/vvc_ps.c

```diff
@@ -246,7 +247,7 @@ static AVBufferRef *sps_alloc(const H266RawSPS *rsps, AVBufferRef *rsps_buf, voi
     return sps_buf;
 
 fail:
-    av_buffer_unref(&sps_buf);
+    ff_refstruct_unref(&sps_buf);
```

Comment:
line 237 and 238, should use ff_refstruct_replace. 
all variables who used ff_refstruct_replace need use ff_refstruct_replace and ff_refstruct_ref too.